### PR TITLE
feat: allow this to work with a cron schedule

### PR DESCRIPTION
**Pull request by garethjevans**
When triggering via a cron schedule we list all PRs against the current repo and evaluate them

### CODE REVIEW

File: cmd/action_test.go [hunk 1]
```diff
@@ -32,27 +32,27 @@ func TestGetLabelerConfigV0(t *testing.T) {
 	}
 
 	expectMatchers := map[string]l.LabelMatcher{
-		"WIP": l.LabelMatcher{
+		"WIP": {
 			Label: "WIP",
 			Title: "^WIP:.*",
-		},
-		"WOP": l.LabelMatcher{
+				},
```
Comment (srvaroa) [inline cmd/action_test.go:38]: Not intentional I guess? (same down to L55)

**Review by srvaroa — COMMENTED** [review]
Minor cosmetic issue, otherwise I think it's ready to merge. Thanks!

File: cmd/action.go [hunk 2]
```diff
@@ -45,9 +45,28 @@ func main() {
...
+
+		prs, _, err := gh.PullRequests.List(context.Background(), owner, repoName, &github.PullRequestListOptions{})
+		if err != nil {
+			return
+		}
+
+		for _, pr := range prs {
+			err = l.ExecuteOn(pr)
+			if err != nil {
+				return
```
Comment (srvaroa) [inline cmd/action.go:62]: Would it make sense to output the error and try on the next PRs?

Comment (garethjevans) [reply cmd/action.go]: Good spot, done

**Comment by garethjevans** [comment]
Hi @srvaroa, is there anything else you'd like me to do on this change?

**Review by srvaroa — APPROVED** [review]
Hi, sorry I had missed merging it.  Thanks for contributing this!

**Pull request merged**